### PR TITLE
Send contacts to publishing api

### PR DESCRIPTION
**Pull request by steventux**
https://trello.com/c/JxKRVn6c/731-export-whitehall-contacts-to-publishing-api-medium

### CODE REVIEW

File: app/presenters/publishing_api_presenters/contact.rb [hunk 1]
```diff
@@ -0,0 +1,121 @@
+require_relative "../publishing_api_presenters"
+
+class PublishingApiPresenters::Contact
```
Comment (danielroseman) [inline app/presenters/publishing_api_presenters/contact.rb]: Could this inherit from Item, like the other presenters do? If the differences are too great, it would be good to explain that somewhere.

Comment (steventux) [inline app/presenters/publishing_api_presenters/contact.rb]: I've amended [this commit](https://github.com/alphagov/whitehall/pull/2632/commits/1f884ba2139aec91ad87bc3eed664f359204ad13) to explain why this change doesn't follow the `ItemPresenter` inheritance pattern TL;DR, because contacts do not have a `base_path`.

**Comment by danielroseman** [comment]
:+1:
